<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Escama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>243</width>
        <height>663</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>243</width>
        <height>663</height>
       </size>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>243</width>
         <height>331</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>243</width>
         <height>331</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>243</width>
         <height>331</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <widget class="QLabel" name="lblCapCam">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>223</width>
          <height>311</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>311</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>311</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>243</width>
         <height>331</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>243</width>
         <height>331</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>243</width>
         <height>331</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <widget class="QLabel" name="lblCapCard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>223</width>
          <height>311</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>311</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>311</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>575</width>
        <height>663</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="lyTableCards"/>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="title">
             <string>Opciones</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>65</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>65</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Expansi√≥n:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbSets">
                  <property name="font">
                   <font>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Tiempo de captura:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                 <number>6</number>
                </property>
                <property name="verticalSpacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="2" column="1">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Segundos</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QSpinBox" name="sboxSegs">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                  <property name="minimum">
                   <number>6</number>
                  </property>
                  <property name="maximum">
                   <number>99</number>
                  </property>
                  <property name="value">
                   <number>13</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QFormLayout" name="formLayout_3">
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Precisi√≥n de captura:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="sboxCapAcc">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                  <property name="minimum">
                   <number>400</number>
                  </property>
                  <property name="maximum">
                   <number>4000</number>
                  </property>
                  <property name="value">
                   <number>3000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gpbox_Cardmarket">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Cardmarket</string>
            </property>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>120</width>
               <height>150</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>150</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnSendToCardmarket">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>VENDER</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelSelectedRows">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>BORRAR
SELECIONADOS</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelAllCards">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>BORRAR
TODAS</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="btnInit">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="text">
             <string>INICIAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdTimer">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>2</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="btnConfig">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="text">
             <string>CONF.
ESCAMA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExit">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>85</height>
             </size>
            </property>
            <property name="text">
             <string>SALIR</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
